components/cover/basic/src/index.scss

```diff
@@ -0,0 +1,70 @@
+@import '~@schibstedspain/theme-basic/lib/index';
```

Comment:
I think the opposite. I think it is a bad practice because 1)you might be hiding a possible risk 2)you might be overriding an obsolete rule in the wrong place 3)you might be damaging a future attempt to improve linting rules or linting reports.